.github/workflows/nightly.yml

```diff
@@ -121,6 +121,8 @@ jobs:
         run: |
           cp ${{ github.workspace }}/.github/workflows/nightly_notes.md "${RUNNER_TEMP}/nightly_notes.md"
           cd nightly_release
+          echo '' >> "${RUNNER_TEMP}/nightly_notes.md"
+          echo '```' >> "${RUNNER_TEMP}/nightly_notes.md"
```

Comment:
This is a manual change to have a nice nightly_notes.md in the repo

Reply:
Why does it need ``` in the end?